rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -153,44 +153,41 @@ public void startNode(String[] args) throws Exception {
 
         if (rskSystemProperties.isRpcEnabled()) {
             logger.info("RPC enabled");
-            enableRpc();
+            startRPCServer();
         }
         else {
             logger.info("RPC disabled");
         }
 
+        if (rskSystemProperties.isPeerDiscoveryEnabled()) {
+            udpServer.start();
+        }
+
         if (rskSystemProperties.isSyncEnabled()) {
             syncPool.updateLowerUsefulDifficulty();
             syncPool.start();
             if (rskSystemProperties.waitForSync()) {
-                waitRskSyncDone(rsk);
+                waitRskSyncDone();
             }
         }
 
-        if (rskSystemProperties.minerServerEnabled()) {
+        if (rskSystemProperties.isMinerServerEnabled()) {
             minerServer.start();
 
-            if (rskSystemProperties.minerClientEnabled()) {
+            if (rskSystemProperties.isMinerClientEnabled()) {
                 minerClient.mine();
             }
         }
 
-        if (rskSystemProperties.peerDiscovery()) {
-            enablePeerDiscovery();
-        }
-    }
-
-    private void enablePeerDiscovery() {
-        udpServer.start();
     }
 
-    private void enableRpc() throws InterruptedException {
+    private void startRPCServer() throws InterruptedException {
         web3Service = web3Factory.newInstance();
         web3Service.start();
         JsonRpcWeb3ServerHandler serverHandler = new JsonRpcWeb3ServerHandler(web3Service, rskSystemProperties.getRpcModules());
         JsonRpcWeb3FilterHandler filterHandler = new JsonRpcWeb3FilterHandler(rskSystemProperties.corsDomains());
         new JsonRpcNettyServer(
-            rskSystemProperties.rpcHost(),
+            rskSystemProperties.getBindAddress(),
```

Comment:
with this change JSON RPC listens by default in public interfaces again. JSON RPC must listen by default only on localhost.

Reply:
Ok, it will allow to define a specific bind for rpc and a different default too